com.unity.render-pipelines.universal/Runtime/UniversalRenderPipeline.cs

```diff
@@ -640,7 +640,8 @@ static void SetSupportedRenderingFeatures()
                 lightProbeProxyVolumes = false,
                 motionVectors = false,
                 receiveShadows = false,
-                reflectionProbes = true,
+                reflectionProbes = false,
+                reflectionProbesBlendDistance = true,
```

Comment:
Move this behind a define based on the trunk PR. This should allow the changes to the package work independently from the trunk version.

Reply:
Define won't be necessary because we are in core now.